pkg/controllers/repositoryserver/server.go

```diff
@@ -65,6 +65,9 @@ func (h *RepoServerHandler) startRepoProxyServer(ctx context.Context) (err error
 			Background:       true,
 		},
 	)
+
+	cmd = append([]string{"tail", "-v", "-F", "/tmp/kopia-log/cli-logs/latest.log", "&"}, cmd...)
```

Comment:
@aaron-kasten

IIUC, this is tailing the logs as part of the same KubeExec call.
What about redirecting the `kopia server` output to a file that can be read or tailed from the container's **entry point** ?

Note: `latest.log` is a symlink, does that affect `tails -F` behavior?

Questions:
- Are kubeExec commands executed via a shell? Pardon my ignorance, this is for my edification.
- What are the effects of tailing the log output as part of the kubeExec? what happens when the connection is interrupted?